AirCasting/Utils/Strings.swift

```diff
@@ -1,302 +1,436 @@
-        static let buildText = LocalizedStringKey("build").stringKey
-        static let betaBuild = LocalizedStringKey("Beta build").stringKey
-        static let debugBuild = LocalizedStringKey("Debug build").stringKey
-    }
-
+        static let title: String = NSLocalizedString("Settings",
+                                                     comment: "")
+        static let myAccount: String = NSLocalizedString("My Account",
+                                                         comment: "")
+        static let crowdMap: String = NSLocalizedString("Contribute to CrowdMap",
+                                                        comment: "")
+        static let crowdMapDescription: String = NSLocalizedString("Data contributed to the CrowdMap is publicly available at aircasting.org", comment: "")
+        static let disableMapping: String = NSLocalizedString("Disable Mapping",
+                                                              comment: "")
+        static let disableMappingDescription: String = NSLocalizedString("Turns off GPS tracking & session syncing. Use \"Share file\" to retrieve your measurements via email.",
+                                                                         comment: "")
+        static let temperature = NSLocalizedString("Temperature Units",
+                                                   comment: "")
+        static let celsiusDescription = NSLocalizedString("Use Celsius",
+                                                          comment: "")
+        static let backendSettings: String = NSLocalizedString("Backend settings",
+                                                               comment: "")
+        static let settingsHelp: String = NSLocalizedString("Help",
+                                                            comment: "")
+        static let hardwareDevelopers: String = NSLocalizedString("Hardware developers",
+                                                                  comment: "")
+        static let about: String = NSLocalizedString("About AirCasting",
+                                                     comment: "")
+        static let keepScreenTitle = NSLocalizedString("Keep screen on",
+                                                       comment: "")
+        static let clearSDTitle = NSLocalizedString("Clear SD card",
+                                                    comment: "")
+        static let appInfoTitle = NSLocalizedString("AirCasting App v",
+                                                    comment: "")
+        static let buildText = NSLocalizedString("build",
+                                                 comment: "")
+        static let betaBuild = NSLocalizedString("Beta build",
+                                                 comment: "")
+        static let debugBuild = NSLocalizedString("Debug build",
+                                                  comment: "")
```

Comment:
I think we can remove beta/debug only strings from localization at all. It's our internal thing and would be a waste of money or/and time to localize those.